src/components/HelpPage/Faqs/index.js

```diff
@@ -1,36 +1,89 @@
-import React, { Component } from 'react'
+import React,{useState,useCallback} from 'react'
 import '../help.css';
 import faqimg from './faq.png';
 
-export default class Faqs extends Component {
-    render() {
+const initialList=[
+    {
+        id:1,
+        question:"Do I need to login to use the application?",
+        answer:"You need to login to create a post or upvote, downvote or comment on anything. Without logging in you can view the general feed.",
+        isShow:false
+    },
+    {
+        id:2,
+        question:"What should I do if I find any post abusive or against someone?",
+        answer:"You can report the post and it will be taken into consideration by the admin if it is necessary to report the post.",
+        isShow:false
+    },
+    {
+        id:3,
+        question:"Can I post something which is articulated by me?",
+        answer:"No, you just have to post the link of an article with its title, any relevant image and its corresponding tags."
```

Comment:
@rohan2734 You have added `isShow` in first two questions only. Add that in all the questions.